msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -51,6 +51,36 @@ Even if you intend your app to only run in Teams (and not Microsoft 365 app and
 
 Once you're able, the next step is to [update existing application code](#2-update-teamsjs-references) with the changes described in this article. In the meantime, the v.1 to v.2 API translation layer provides backwards compatibility, ensuring your existing Teams app continues to work in TeamsJS version 2.0.
 
+To implement Teams app-specific logic, include the following code snippet to your app that initializes the TeamsSDK and runs in Teams platform exclusively.
+
+```js
+import 'https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js';
+
+// Ensure that the Teans SDK is initialized once no matter how often this is called
+let teamsInitPromise;
+export function ensureTeamsSdkInitialized(){
```

Comment:
Could we rename *ensureTeamsSdkInitialized* to `ensureTeamsJSInitialized` (or something similar)?

Reply:
Done.